lib/board.rb

```diff
@@ -0,0 +1,74 @@
+class Board
+  attr_accessor :board, :booked_slots
+
+  def initialize
+    @board = [[1, 2, 3], [4, 5, 6], [7, 8, 9]]
+    @booked_slots = []
+  end
+
+  def mark_on_board(move, player_mark)
+    (0..@board.length - 1).each do |i|
+      (0..@board.length - 1).each do |j|
+        @board[i][j] = player_mark if @board[i][j] == move
+      end
+    end
+  end
+
+  def display_board
+    (0..@board.length - 1).each do |i|
+      (0..@board.length - 1).each do |j|
+        print @board[i][j].to_s + ' '
```

Comment:
- [ ] No puts/print and get.chomp outside main.rb file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png)) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))